app/components/lg-lion-search.js

```diff
@@ -58,8 +60,14 @@ export default Ember.Component.extend({
 
   actions: {
     search: function() {
-      var params = this.get('params');
-      this.sendAction('action', params);
+      var params = this.get('params'),
+          store = this.get('store'),
+          component = this,
+          searchModel = this.get('searchModel');
+
+      store.find(searchModel, params).then(function(results){
+        component.sendAction('action', results);
+      });
```

Comment:
we'll definitely need to handle errors, too. A generic "there was an error message" is probably sufficient.